database/seeders/ApiEndpointsSeeder.php

```diff
@@ -7010,6 +7010,21 @@ private function seedSummitEndpoints()
                     IGroup::SummitRegistrationAdmins,
                 ]
             ],
+            [
+                'name' => 'send-sponsor-promo-codes-mail',
+                'route' => '/api/v1/summits/{id}/promo-codes/all/send',
```

Comment:
this is not correct
SDS saids
PUT api/v1/summits/{id}/sponsors/all/promo-codes/all/send